finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,9 @@
+for( let i = 10; i > 0; i--){
+  if(i == 1){
```

Comment:
Usually, developers try to stick to i++, though this does work as well. In this case, you'd need to do extra math to have your values, so totally understand using i--. The next step for this challenge would be to try to leverage a function to make this work for any number of frogs.